.github/workflows/pwsh.yaml

```diff
@@ -6,7 +6,7 @@ on:
     paths: ['pwsh_module/*']
   pull_request:
     type: [opened, reopened, edited]
-    paths: ['pwsh_module/*']
+    paths: ['pwsh_module/*', 'lib/bolt/bolt_option_parser.rb']
```

Comment:
This github action should have failed a previous PR because we check that the 'correct' number of parameters are generated for each command (as a safety measure) and we added `envvars` to `bolt command`. I think this happened because I did not put in a check for the `bolt_option_parser.rb` file. I don't know a way of testing this other than commiting and checking later. 

@lucywyman do you think this will catch when we add params to bolt?

Reply:
It should, that file should get touched anytime we modify a command-line command.